tests/llms/test_llms_faq-generation_tgi.sh

```diff
@@ -0,0 +1,114 @@
+﻿#!/bin/bash
```

Comment:
![image](https://github.com/user-attachments/assets/ad8312c4-ada6-4002-8f04-1368c16c9dd6)
Please use the same integration name, langchain or tgi.